src/ImageGallery.js

```diff
@@ -293,14 +293,33 @@ class ImageGallery extends React.Component {
     return 0;
   }
 
+  getThumbnailPositionClassName(thumbnailPosition) {
+    // get the specific thumbnailPosition className
+    const leftClassName = 'image-gallery-left';
+    const rightClassName = 'image-gallery-right';
+
+    switch (thumbnailPosition) {
+      case ('left'):
+        thumbnailPosition = ` ${leftClassName}`;
+        break;
+      case ('right'):
+        thumbnailPosition = ` ${rightClassName}`;
+        break;
+      default:
```

Comment:
Although we're not specifically styling bottom and top thumbnails, Id like to keep the classname so there are consistent class indicators.

lets add 

```
case 'bottom':
case 'top': 
```